BlockSettleUILib/BSTerminalMainWindow.cpp

```diff
@@ -208,17 +208,22 @@ void BSTerminalMainWindow::onInitWalletDialogWasShown()
 
 void BSTerminalMainWindow::onAddrStateChanged()
 {
-   bool canSubmitAuthAddr = (userType_ == bs::network::UserType::Trading)
+   const bool canSubmitAuthAddr = (userType_ == bs::network::UserType::Trading)
          || (userType_ == bs::network::UserType::Dealing);
 
-   if (allowAuthAddressDialogShow_ && authManager_ && authManager_->HasAuthAddr() && authManager_->isAllLoadded()
-      && !authManager_->isAtLeastOneAwaitingVerification() && canSubmitAuthAddr) {
+   const bool needSignAuthAddress = authManager_ && authManager_->HasAuthAddr() && authManager_->isAllLoadded()
+      && !authManager_->isAtLeastOneAwaitingVerification();
+
+   const auto& tradeSettings = authManager_->tradeSettings();
+
+   if (allowAuthAddressDialogShow_ && authManager_ && tradeSettings && needSignAuthAddress && canSubmitAuthAddr) {
       allowAuthAddressDialogShow_ = false;
       BSMessageBox qry(BSMessageBox::question, tr("Authentication Address"), tr("Create Authentication Address")
          , tr("The Authentication Address verifies you as a Participant in our trading network. It is required for Spot XBT (bitcoin) trading.\n\n"
-              "Submitted Authentication Addresses are limited to one (1) bitcoin per trade.\n\n"
-              "After three (3) trades, BlockSettle will automatically fund your Authentication Address with 1,000 satoshis. Once validated, other Participants may independently verify you on-chain as a Participant of BlockSettle, and the trade limit is removed.\n\n"
-              "Create Authentication Address now?\n"), this);
+              "Submitted Authentication Addresses are limited to %1 bitcoin per trade.\n\n"
```

Comment:
I think that xbtTier1Limit is delivered in satoshis. What amount is displayed here?

Reply:
Nice catch! changed it to double. Thank you